core/mutation/mutation.go

```diff
@@ -131,6 +133,10 @@ func (s *Server) GetMutations(ctx context.Context, in *tpb.GetMutationsRequest)
 	}, nil
 }
 
+func (s *Server) GetMutationsStream(in *tpb.GetMutationsRequest, stream spb.MutationService_GetMutationsStreamServer) error {
```

Comment:
I think we can remove this API from here. `MutationService_GetMutationsStreamServer` is in `impl/proto/mutation_v1_service` and cannot be a dependency here.